publish.yml

```diff
@@ -0,0 +1,77 @@
+# This workflow shows how to use the publish and publish-on-error attributes of a task.
+# If the task succeeds, any variables declared in the publish block can be accessed by all downstream tasks.
+# If the task fails, any variables declared in the publish-on-error block can be accessed by all downstream tasks.
+# In this example, the following variables are published by task_1 and task_2:
+#   - job_id
+#   - run_id
+#   - job_name
+#   - started_at
+#   - finished_at
+#   - status
+# These variables are then accessed by the task labeled log_task.
+
+# Note: The use of the anchor (&) is not required to use the publish attribute.
+# It's used here to keep the code D.R.Y. That is, instead of declaring the same variables
+# again in the publish-on-error block, we dereference the variables using an asterisk (*).
+
+version: '2.0'
+workflow:
+  tasks:
+    task_1:
+      description: Some description of task 1
+      action: civis.scripts.python3
+      input:
+        source: |
+          import time
+
+          time.sleep(5)
+          print('Task 1')
+          time.sleep(5)
```

Comment:
Would it be helpful to folk to have this script create a JSON output and publish that, or is that too confusing? Maybe it does make sense to focus on publish here, and we have another example of a JSON output that they can combine.